data_structures/queue/priority_queue.py

```diff
@@ -0,0 +1,248 @@
+"""
+Pure Python implementation of Priority Queue using lists
+"""
+
+
+class FixedPriorityQueue:
+    """
+    In a Priority Queue the elements are entred as an when the come
+    But while removing or deleting an element the highest priority element is deleted in FIFO fashion
+    Here the lowest integer has the highest priority.
+    Example:
+    priority(0) > priority(5)
+    priority(16) > priority(32)
+    Here as an example I have taken only 3 priorities viz. 0, 1, 2
+    0 Being the highest priority and 2 being the lowest
+    You can change the priorities as per your need
+
+    Examples
+    >>>FPQ = FixedPriorityQueue()
```

Comment:
```suggestion
    >>> FPQ = FixedPriorityQueue()
```
The space is significant.  Without it, your tests will _always_ pass.